src/dev/impl/DevToys/ViewModels/Tools/Text/XmlValidator/XmlValidatorToolViewModel.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
```

Comment:
Please add `#nullable enable` at the top of the file.

Reply:
Was added.